ctyparser/bigcty.py

```diff
@@ -92,32 +114,56 @@ def import_dat(self, dat_file: Union[str, os.PathLike]) -> None:
         with dat_file.open("r") as file:
             cty_dict = dict()
 
+            # get the version from the file, set filepointer to beginning of file afterwards
             ver_match = re.search(self.regex_version_entry, file.read())
             self._version = ver_match.group(1) if ver_match is not None else ""
             file.seek(0)
 
+            # variable to store last DXCC introduction with countryname, zones, etc. for further use
```

Comment:
this is wrong. `last` stores the previous prefix *only*. A better fix would be to make the variable name more descriptive, but that is for another time and PR.

```suggestion
            # stores the previous prefix for the next iteration
```